src/classes/hotel.js

```diff
@@ -0,0 +1,110 @@
+import Customer from './customer';
+import Room from './room';
+import Booking from './booking';
+import roomsData from '../../test/test-data/room-data';
+
+class Hotel {
```

Comment:
SUPER SMART!!!!  Makes so much more sense to have these properties and methods 'living' here on a Hotel class.